docs/markdown/Qt5-module.md

```diff
@@ -63,6 +63,7 @@ needs to be specified manually, while QML plugins are automatically detected fro
 
 The method takes the following keyword arguments:
 - `qresources`, rcc source file used for scanning for QML plugins
+- `qmlfiles`, sources files used for scanning for QML plugins
```

Comment:
It's slightly clearer to document that these are QML files rather than just "source files":

```diff
+ - `qmlfiles`, QML source files to scan for QML plugins
```